torch/nn/modules/module.py

```diff
@@ -1574,8 +1580,16 @@ def _call_impl(self, *args, **kwargs):
 
     __call__ : Callable[..., Any] = _call_impl
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
```

Comment:
```suggestion
        state = super().__getstate__()
```

Reply:
Actually just reverted this - getting this 

```
  File "/home/ubuntu/pytorch/torch/nn/modules/module.py", line 1584, in __getstate__
    state = super().__getstate__()
AttributeError: 'super' object has no attribute '__getstate__'
```

Reply:
Ho wow TIL!